docs/api/insert.md

```diff
@@ -246,7 +246,7 @@ Please be careful that the size of the request exceed the limit.
     |field|type|label|required|desc.|
     |:---:|:---|:---|:---:|:---|
     |skip_strict_exist_check|bool| | | check the same vector is already inserted or not.<br>the ID should be unique if the value is `true`|
-    |timestamp|int64| | | the timestamp of the vector inserted.<br>if it is N/A, the current time will be used.
+    |timestamp|int64| | | the timestamp of the vector inserted.<br>if it is N/A, the current time will be used |
```

Comment:
Existence or not of "." is not consistent.

Reply:
should we use `.` for all sentences?

Reply:
@kmrmt what do you think? I think not using `.` is fine